src/client/tile.cpp

```diff
@@ -122,9 +122,11 @@ std::string getImagePath(std::string path)
 
 	Utilizes a thread-safe cache.
 */
-std::string getTexturePath(const std::string &filename)
+std::string getTexturePath(const std::string &filename, bool *is_base_pack)
 {
 	std::string fullpath;
+	if (is_base_pack)
+		*is_base_pack = false;
```

Comment:
There's no way to tell the source of the texture (in a sane way) after caching the path.
It's also not needed, though since this `is_base_pack` is only passed when initializing the textures the first time.

Reply:
Please note this face in a comment.

Reply:
Done